node/node.go

```diff
@@ -125,25 +125,31 @@ const (
 )
 
 func GetCommand() *cobra.Command {
+	conf := config.MainnetConfig()
+	var configPath *string
 	c := &cobra.Command{
 		Use:   "node",
 		Short: "start node",
 		Run: func(c *cobra.Command, args []string) {
-			conf, err := loadConfig(c)
-			if err != nil {
+			preset := conf.Preset // might be set via CLI flag
+			if err := loadConfig(&conf, preset, *configPath); err != nil {
 				log.With().Fatal("failed to initialize config", log.Err(err))
 			}
+			// apply CLI args to config
+			if err := c.ParseFlags(args); err != nil {
+				fmt.Println("an error has occurred while parsing flags:", err)
```

Comment:
With the suggested change above this becomes:

```suggestion
				return fmt.Errorf("an error has occurred while parsing flags: %w", err)
```

Same with all the `log.With().Fatal` calls.